pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -364,6 +364,11 @@ func (e *etcd) Deploy(ctx context.Context) error {
 			containerPolicyOff = autoscalingv1beta2.ContainerScalingModeOff
 		)
 
+		scaleDownUpdateMode := e.hvpaConfig.ScaleDownUpdateMode
+		if scaleDownUpdateMode == nil {
+			scaleDownUpdateMode = pointer.StringPtr(hvpav1alpha1.UpdateModeMaintenanceWindow)
```

Comment:
We are defaulting the value twice, here and in `pkg/operation/botanist/etcd.go`? Same question for KCM and KAS (defaulting in chart).